LandLord-legacy/src/main/java/biz/princeps/landlord/manager/MobManager.java

```diff
@@ -86,9 +86,9 @@ public IMob valueOf(String name) {
 
     public static class Mob implements IMob {
 
-        EntityType t;
-        int egg;
-        String permission;
+        final EntityType t;
+        final int egg;
+        final String permission;
```

Comment:
Why does this class has setter and package private fields? Shouldn't these be private?

Reply:
Ni idea 😅